### PR TITLE
OSDOCS-2031: Adding known issue for customizing console routes

### PR DESCRIPTION
**Pull request by bergerhoffer**
https://issues.redhat.com/browse/OSDOCS-2031

Preview: https://deploy-preview-33783--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-8-release-notes?utm_source=github&utm_campaign=bot_dp#ocp-4-8-known-issues

### CODE REVIEW

**Comment by ahardin-rh** [comment]
Looks great! :shipit:

**Comment by bergerhoffer** [comment]
@quarterpin Can you please QE review this?

Preview: https://deploy-preview-33783--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-8-release-notes?utm_source=github&utm_campaign=bot_dp#ocp-4-8-known-issues

**Comment by quarterpin** [comment]
It looks good to me but I would like to ask @yanpzhan to have a glance at this as well since this change is specifically for console route configuration. Thanks!

File: release_notes/ocp-4-8-release-notes.adoc [hunk 1]
```diff
@@ -1215,7 +1217,9 @@ This script removes unauthenticated subjects from the following cluster role bin
...
+* An Open Virtual Network (OVN) bug causes persistent connectivity issues with Octavia load balancers. When Octavia load balancers are created, OVN might not plug them into some Neutron subnets. These load balancers might be unreachable for some of the Neutron subnets. This problem affects Neutron subnets, which are created for each OpenShift namespace, at random when Kuryr is configured. As a result, when this problem occurs the load balancer that implements OpenShift `Service` objects will be unreachable from OpenShift namespaces affected by the issue. Because of this bug, {product-title} 4.8 deployments that use Kuryr SDN are not recommended on {rh-openstack-first} 16.1 with OVN and OVN Octavia configured until the bug is fixed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1937392[*BZ#1937392*])
+
+* When either of the console's routes (`console` or `download`) are customized, the Console Operator does not properly update the `Ingress` resource with the `componentRoutes` conditions. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1954148[*BZ#1954148*])
```
Comment (awgreene) [inline release_notes/ocp-4-8-release-notes.adoc:1222]: the Console Operator does not properly update the `Ingress` resource with the `componentRoutes` conditions. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1954148[*BZ#1954148*])
```suggestion
* The Console Operator does not properly update the `Ingress` resource with the `componentRoutes` conditions for  either of the console's routes (`console` or `download`). (link:https://bugzilla.redhat.com/show_bug.cgi?id=1954148[*BZ#1954148*])
```

Comment (awgreene) [reply release_notes/ocp-4-8-release-notes.adoc:1222]: Also - [it looks like](https://deploy-preview-33783--osdocs.netlify.app/openshift-enterprise/latest/web_console/customizing-the-web-console.html#customizing-the-download-route_customizing-web-console) the "download route" is called the "downloads".

**Pull request merged**